kuiper-tests/JsonFileSolarSystemRepositoryShould.cs

```diff
@@ -44,170 +48,130 @@ public void NotFailIfJsonFileExist()
-            var jsonString = "{'position':1,'name':'Sol','distance':0,'velocity':0,'originDegrees':0,'type':'Moon','satellites':[]}";
+            var repo = new JsonFileSolarSystemRepository(_testDataFilePath);
 
             //Act
-            var repo = new JsonFileSolarSystemRepository(new FileInfo(file));
-            var bodies = repo.GetBodiesFromJson(jsonString);
+            var bodies = repo.GetSolarSystem();
 
             //Assert
-            Assert.IsType<Moon>(bodies.SingleOrDefault());
-
-            File.Delete(file);
+            Assert.True(bodies.OfType<Moon>().Count() == 1);
         }
 
         [Fact]
         public void CreateGasGiantBodyTypeFromJson()
         {
             //Arrange
-            var file = Path.GetTempFileName();
-            var jsonString = "{'position':1,'name':'Sol','distance':0,'velocity':0,'originDegrees':0,'type':'GasGiant','satellites':[]}";
+            var repo = new JsonFileSolarSystemRepository(_testDataFilePath);
 
             //Act
-            var repo = new JsonFileSolarSystemRepository(new FileInfo(file));
-            var bodies = repo.GetBodiesFromJson(jsonString);
+            var bodies = repo.GetSolarSystem();
 
             //Assert
-            Assert.IsType<GasGiant>(bodies.SingleOrDefault());
-
-            File.Delete(file);
+            Assert.True(bodies.OfType<GasGiant>().Count() == 2);
         }
 
         [Fact]
         public void CreateDwarfPlanetBodyTypeFromJson()
         {
             //Arrange
-            var file = Path.GetTempFileName();
-            var jsonString = "{'position':1,'name':'Sol','distance':0,'velocity':0,'originDegrees':0,'type':'DwarfPlanet','satellites':[]}";
+            var repo = new JsonFileSolarSystemRepository(_testDataFilePath);
 
             //Act
-            var repo = new JsonFileSolarSystemRepository(new FileInfo(file));
-            var bodies = repo.GetBodiesFromJson(jsonString);
+            var bodies = repo.GetSolarSystem();
 
             //Assert
-            Assert.IsType<DwarfPlanet>(bodies.SingleOrDefault());
-
-            File.Delete(file);
+            Assert.True(bodies.OfType<DwarfPlanet>().Count() == 1);
         }
 
         [Fact]
         public void SetCorrectOrbitRadius()
         {
             //Arrange
-            var file = Path.GetTempFileName();
-            var jsonString = "{'position':1,'name':'Sol','distance':42,'velocity':0,'originDegrees':0,'type':'DwarfPlanet','satellites':[]}";
+            var repo = new JsonFileSolarSystemRepository(_testDataFilePath);
 
             //Act
-            var repo = new JsonFileSolarSystemRepository(new FileInfo(file));
-            var bodies = repo.GetBodiesFromJson(jsonString);
+            var bodies = repo.GetSolarSystem();
 
             //Assert
-            Assert.Equal(42, bodies.SingleOrDefault().OrbitRadius);
-
-            File.Delete(file);
+            //Assert.True(bodies.Single(x => x.Name == "Earth").OrbitRadius);
```

Comment:
Should this really be commented?
There's no Assert in this test?
```suggestion
            Assert.Equal(1d, bodies.Single(x => x.Name == "Earth").OrbitRadius);
```

Funnily, this test would have revealed that something was wrong with the mapping of that distance->orbitRadius property :)

Reply:
heh - Yeah, but maybe you don't need a single testcase for every property? I would probably combine them to load the json once and then assert different things on the object model?

Reply:
Yea, that's what I initially did but when the test failed, it didn't show me which assert failed.
However, I have later learnt, that you can name the asserts, or even define the error message, to get around that issue.

Reply:
But, unless you want to actually refactor the tests, we should probably just enable this again?

Reply:
Refactored this myself. Nevermind.